OpenRA.Mods.Common/Activities/Parachute.cs

```diff
@@ -20,15 +20,17 @@ public class Parachute : Activity
 		readonly IPositionable pos;
 		readonly ParachutableInfo para;
 		readonly WVec fallVector;
+		readonly Actor ignored;
 
 		WPos dropPosition;
 		WPos currentPosition;
 		bool triggered = false;
 
-		public Parachute(Actor self, WPos dropPosition)
+		public Parachute(Actor self, WPos dropPosition, Actor ignore)
```

Comment:
I suggest renaming this to `ignoreActor` for consistency with the other traits, and you can default it to null so that you don't need to modfy EjectOnDeath or Paradrop.